File: C/Assignment14/Program5.c
///////////////////////////////////////////////////////////////////////////////////////////////////////
//  
// File name :   Program5.c
// Description : Program which accept number of rows & columns from user & display below pattern
// Author :      Pallavi Omprakash Malewar
// Date :        06/07/2026
//
///////////////////////////////////////////////////////////////////////////////////////////////////////


/*

Input :     iRow = 5      iCol = 5

Output :    1   2   3   4   
            2   3   4   5
            3   4   5   6
            4   5   6   7

*/


#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    for(int i = 0 ; i < iRow ; i++)
    {
        for(int j = 0 ; j < iCol ; j++)
        {
            printf("%d ", (i + j + 1));
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows & columns  :");
    scanf("%d %d" , &iValue1 , &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: JAVA/Assignment 13/Program5.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Write a Java Program to find transpose of a matrix.             //
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 6 Aug 2026                                                                   //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////


import java.util.Scanner;

public class Program5
{
    public static void main(String A[])
    {
        int row, col;

        Scanner sobj = new Scanner(System.in);

        // Read number of rows and columns
        System.out.print("Enter Number of Rows : ");
        row = sobj.nextInt();

        System.out.print("Enter Number of Columns : ");
        col = sobj.nextInt();

        // declare two dimensional array (matices)
        int a[][] = new int[row][col];

        // Read elements of Matrix Arr
        System.out.println("Enter elements of matrix A : ");
        for(int i = 0 ; i < row ; i++)
        {
            for(int j = 0 ; j < col ; j++)
            {
                System.out.print("Element [ " + (i + 1) + "," + (j + 1) + " ] ? ");
                a[i][j] = sobj.nextInt();
            }
        }

        // Print matrix A
        System.out.println("Matrix A : ");
        for(int i = 0 ; i < row ; i++)
        {
            for(int j = 0 ; j < col ; j++)
            {
                System.out.print(a[i][j] + "\t");
            }
            System.out.println();
        }

        // Print matrix B
        System.out.println("::: Transpose Matrix ::: ");
        for(int i = 0 ; i < col ; i++)
        {
            for(int j = 0 ; j < row ; j++)
            {
                System.out.print(a[j][i] + "\t");
            }
            System.out.println();
        }
    }
}